CMakeLists.txt

```diff
@@ -23,16 +23,21 @@ endif()
 
 include(cmake/CPM.cmake)
 
-# PackageProject.cmake will be used to make our target installable
-CPMAddPackage("gh:TheLartians/PackageProject.cmake@1.8.0")
+# PackageProject.cmake will be used to make our target installable XXX
+# CPMAddPackage("gh:TheLartians/PackageProject.cmake@1.13.0")
+include(cmake/PackageProject.cmake)
```

Comment:
@TheLartians NOTE: The original `PackageProject.cmake` can not use with `FILE_SET HEADERS`!